src/chrome/breakOnLoadHelper.ts

```diff
@@ -57,6 +58,10 @@ export class BreakOnLoadHelper {
         return this._chromeDebugAdapter.scriptsById.get(scriptId).url;
     }
 
+    public async setBrowserVersion(version: Version): Promise<void> {

```

Comment:
Please add some comments on why we care about the version number.  Please open (and document here) a Chromium bug if you haven't already on this.

Reply:
Done:
        // On version 69 Chrome stopped sending an extra event for DOM Instrumentation: See https://bugs.chromium.org/p/chromium/issues/detail?id=882909
        // On Chrome 68 we were relying on that event to make Break on load work on breakpoints on the first line of a file. On Chrome 69 we need an alternative way to make it work.